### ISSUE TITLE
Input Created by Ranged Date Picker remains on page after release

### ISSUE THREAD

**Issue by hvlawren** [carbon-design-system/carbon#806]
## Detailed description

This is an issue with the ranged date picker. It creates a visually hidden input that seems to be used by flatpickr in the ranged date picker. This input lacks the aria-hidden tag which would cause screen readers to skip it, and remains on the page after navigating away from the ranged date picker.

> Is this issue related to a specific component? 

Ranged date picker

> What did you expect to happen? What happened instead? What would you like to see changed?

Visually hidden elements should be hidden from screen readers unless they convey information specific to them. This is not hidden.

The input should also be removed from the DOM after the date picker is released. It is not removed.

> What browser are you working in? 

Tested in Chrome and Safari.

> What version of the Carbon Design System are you using? 

8.18
